transformer_engine/pytorch/csrc/extensions/quantizer.cpp

```diff
@@ -7,9 +7,9 @@
 #include <pybind.h>
 
 #include "common.h"
+#include "common/common.h"
```

Comment:
@ptrendx what's the right way to use the function defined in `common` in framework specific dirs like this one?